modules/builds-basic-edit-buildconfig.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+// * builds/basic-build-operations.adoc
+
+[id="builds-basic-edit-buildconfig_{context}"]
+= Editing a BuildConfig
+
+You can use the *Edit BuildConfig* option in the *Builds* view of the *Developer* perspective to edit your build configurations. You can edit a `BuildConfig` using the following options:
+
+* *Form view*: This allows you to edit and configure your `BuildConfig` using the standard form fields and checkboxes.
```

Comment:
```suggestion
* The *Form view* enables you to edit and configure your `BuildConfig` using the standard form fields and checkboxes.
```
s/allows/enables/ to avoid anthropomorphism.